SL/Mailer.pm

```diff
@@ -22,143 +31,122 @@ sub new {
   bless $self, $type;
 }
 
-
 sub send {
-  my ($self, $out) = @_;
-
-  my $boundary = time;
-  my $domain = $self->{from};
-  $domain =~ s/(.*?\@|>)//g;
-  my $msgid = "$boundary\@$domain";
-  $boundary = "SL-$self->{version}-$boundary";
-  
-  $self->{charset} ||= "ISO-8859-1";
-  
-  if ($out) {
-    open(OUT, $out) or return "$out : $!";
-  } else {
-    open(OUT, ">-") or return "STDOUT : $!";
-  }
+  my ($self) = @_;
+
+  #----- CONFIGURATION
+  my $tmpfolder = 'http://domain.net/ledger123/rma/users';
+  my $apikey = 'xkeysib-ad3c502c031275016ba77259cbaad630f58431abfa6eb4938c56e62e7dc64591-UfY14WhVRGN';
+  #----- CONFIG END
```

Comment:
Config will follow?

Reply:
Yes. exactly. This is for the time being while I am testing everything.